run_release.py

```diff
@@ -1138,10 +1138,10 @@ def _api_key(api_key):
         Task(run_blurb_release, "Run blurb release"),
         Task(check_cpython_repo_is_clean, "Checking git repository is clean"),
         Task(prepare_pydoc_topics, "Preparing pydoc topics"),
+        Task(bump_version, "Bump version"),
```

Comment:
Previously, regeneration of ``configure`` was running before ``bump_version`, so it actually "worked", but not as expected :)

Reply:
Yeah, this is fine, but note that this was paired with the "is repository clean" check in the line right after. I'd move that, too.

Reply:
Sure. Check the last commit, please.